### PR TITLE
update couchbase license

### PR DESCRIPTION
**Pull request by xdch47**
Please update couchbase license as upstream did here couchbase/goutils@b496390 in order to remove license doubts.

Refs:
gentoo/gentoo#13906
gentoo/gentoo#12597

### CODE REVIEW

File: vendor/github.com/couchbase/goutils/LICENSE.md [hunk 1]
```diff
@@ -1,47 +1,202 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "{}"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright {yyyy} {name of copyright owner}
```
Comment (xdch47) [inline vendor/github.com/couchbase/goutils/LICENSE.md:189]: upps, seems like upstream forget to substite this with something like ``Copyright (c) 2016-2019 Couchbase, Inc``

**Comment by techknowlogick** [comment]
Thanks for this PR :) Could you update go modules for couchbase dep, and then run `make vendor`? (if not, let me know and I can push those changes to your branch)

**Comment by xdch47** [comment]
sorry just recognised this - maybe this requires to be fixed at couchbase/goutils first?

**Comment by zeripath** [comment]
Personally I'd say it's clear what they mean. The law doesn't work like a program - they clearly intend this to be under Apache 2 with the owner of couchbase as the owner.

File: vendor/github.com/couchbase/goutils/LICENSE.md (hunk 1 above)
Comment (zeripath) [reply vendor/github.com/couchbase/goutils/LICENSE.md:189]: That's just part of the Appendix of how to apply Apache 2 to your work. It's correct to leave that there unchanged.

**Comment by xdch47** [comment]
okay then it's fine

**Comment by xdch47** [comment]
@techknowlogick 
so it actually has be update here first https://gitea.com/macaron/session/src/branch/master/go.mod#L9
right?

**Comment by zeripath** [comment]
ah! yeah that would be where it needs to go. Would you be able to open a PR there?

**Comment by xdch47** [comment]
--> https://gitea.com/macaron/session/pulls/10

**Pull request closed without merging**